meinberlin/apps/kiezradar/models.py

```diff
@@ -170,6 +171,68 @@ def __str__(self):
         return f"kiezradar search profile - {self.name}, disabled {self.disabled}"
 
 
+def full_text_search(
+    search_term: str,
+    search_queryset: Optional[QuerySet[SearchProfile]] = None,
+    language: str = "german",
+) -> QuerySet[SearchProfile]:
+    """
+    Performs a full-text search on the given SearchProfile QuerySet.
+    To be used with postgresql-only database connection.
+
+    Args:
+        search_term (str): The user-provided search input.
+        search_queryset (Optional[QuerySet[SearchProfile]]):
+            A base QuerySet to apply the search on (default: all SearchProfile records).
+        language (str, optional): The PostgreSQL full-text search language configuration. Defaults to "german". If more languages need to be supported than german, the config would need a library like nltk to sanitize multiple languages.
+
+    Returns:
+        QuerySet[SearchProfile]: A queryset of matching search profiles.
+    """
+
+    # Default to all search profiles if no QuerySet is provided
+    search_queryset = search_queryset or SearchProfile.objects.all()
+
+    query = " or ".join(search_term.split())
+    search_query = SearchQuery(query, search_type="websearch", config="german")
```

Comment:
can you replace `"german"` with `language` here? I forgot we are passing it as a parameter.